src/app/authorization/registration/registration.component.html

```diff
@@ -1,129 +1,236 @@
-            <div class="form-text text-danger" *ngIf="Cpwd.errors?.['required'] && (Cpwd.dirty || Cpwd.touched)">
-                confirmnpassword password is Required..
-            </div>
-            <div class="form-text text-danger" *ngIf="Cpwd.errors?.['minlength'] || Cpwd.errors?.['maxlength'] && (Cpwd.dirty || Cpwd.touched)">
-               confirmPassword should be at least 6 and at the max 15 characters long. 
-            </div>
-            <div *ngIf="Cpwd.errors?.['confirmPasswordValidator']">
-                Passsword and Confirm Password didn't match.
-              </div>
-              <div class="form-text text-danger" *ngIf="registrationForm.errors?.['identityRevealed'] && (registrationForm.touched || registrationForm.dirty)">
-                Passsword and Confirm Password didn't match.
+          
+            <label for="cpwd" class="form-label" style="margin-top:15px"
+              >Confirm Password</label
+            >
+            <input
+              type="password"
+              class="form-control"
+              id="cpwd"
+              formControlName="cpwd"
+            />
+            <div
+              class="form-text text-danger"
+              *ngIf="registrationFormValidator['cpwd']?.errors?.['required'] && (registrationFormValidator['cpwd']?.dirty || registrationFormValidator['cpwd']?.touched)"
+            >
+              Confirmn Password is Required..
             </div>
+
+            <div
+              class="form-text text-danger"
+              *ngIf="registrationFormValidator['cpwd']?.errors?.['identityRevealed'] && registrationFormValidator['cpwd']?.touched"
+            >
+              Passsword and Confirm Password didn't match.
             </div>
+          
         </div>
-    </div>
-    <div class="row">
+      </div>
+      <div class="row">
         <div class="gender">
-        <div class="m-1" style="margin-top: 10px;">Gender</div>
-        <div class="wrapper">
+          <div class="m-1" style="margin-top: 10px">Gender</div>
+          <div class="wrapper">
             <div class="form-check form-check-inline">
-                <input type="radio" class="form-check-input" value="m" id="male" formControlName="gender">
-                <label class="form-check-label" for="male">Male</label>
-                
+              <input
+                type="radio"
+                class="form-check-input"
+                value="m"
+                id="male"
+                formControlName="gender"
+              />
+              <label class="form-check-label" for="male">Male</label>
             </div>
             <div class="form-check form-check-inline">
-                <input type="radio" class="form-check-input" value="f" id="female" formControlName="gender">
-                <label class="form-check-label" for="female">Female</label>
+              <input
+                type="radio"
+                class="form-check-input"
+                value="f"
```

Comment:
instead of passing only f please pass full name like female

Reply:
ok i removed it.